Sources/PIALibrary/Client+Preferences.swift

```diff
@@ -481,7 +495,7 @@ extension Client.Preferences {
             useWiFiProtection = true
             trustCellularData = false
             nmtMigrationSuccess = false
-            vpnType = IKEv2Profile.vpnType
+            vpnType = PIAWGTunnelProfile.vpnType
```

Comment:
Same as the other comments: for tvOS we still want IKEv2 to be the default

Reply:
We also need to add any reference to PIAWGTunnelProfile inside the iOS condition since this class is not available on tvOS and there will be a compiler error

Reply:
We need to also have all references to PIAWGTunnelProfile inside the iOS condition since this class is not available on tvOS otherwise it won't compile on tvOS